platformio_orig.ini

```diff
@@ -89,6 +89,7 @@ lib_deps_basic =
     lewisxhe/AXP202X_Library @ ^1.1.3
     geeksville/esp32-micro-sdcard @ ^0.1.1
     256dpi/MQTT @ ^2.4.7
+    https://github.com/dbSuS/libpax#v0.1.1
```

Comment:
https://github.com/dbSuS/libpax#v0.1.1 won't work, while there is no corresponding release in github.
While we don't have a release, we can reference a commit, e.g. https://github.com/dbSuS/libpax#9d0ba8b

Reply:
I set the merge request as a draft because of this as the release was not yet public. I didn't expect this merge request to be merged right away because of this. Sorry for not making that clear.

There should now be a version with the set tag in the public repo. We were not aware of such a fast merge process and also tagged the pull request as draft there for.

The 0.1.1 release is needed for stable wifi+ble switching.